src/test/Fake.Core.UnitTests/Fake.DotNet.Fsdocs.fs

```diff
@@ -7,274 +7,547 @@ open Expecto
 let tests =
     testList
         "Fake.DotNet.Fsdocs.Tests"
-        [ testCase "It append input parameter when its value is overriden"
-          <| fun _ ->
-              let buildParams: Fsdocs.BuildCommandParams =
-                  { Fsdocs.BuildCommandParams.Default with Input = Some("docs-new") }
+        [ testList
+              "build"
```

Comment:
I did split these commands into separate lists. Let me know if I should undo this.